Upstream: Unknown
Source: Daniel Mierswa <impulze@impulze.org>
Reason: GCC errors out since #elif is empty

--- ./toolkit/xre/nsAppRunner.cpp.orig	2009-06-10 05:02:45.000000000 +0200
+++ ./toolkit/xre/nsAppRunner.cpp	2009-06-10 05:02:50.000000000 +0200
@@ -1370,7 +1370,7 @@
   if (NS_FAILED(rv))
     return rv;
 
-#elif
+#else
 #error Oops, you need platform-specific code here
 #endif
 
